.github/workflows/push_production.yaml

```diff
@@ -0,0 +1,31 @@
+# this build is triggered when a new pre-release has been created
+# it creates a new docker build image based on the tag associated
+
+name: push_production
+
+on:
+  release:
+    types:
+      - 'prereleased'
+
+jobs:
+  push:
+    name: push
+    runs-on: ubuntu-latest
+    environment:
+      name: production
+      url: https://search.openverse.engineering
```

Comment:
Should we switch this to a different domain?

Reply:
To ask the same question a different way: If we deploy this as-is now, would this replace our current production site the next time we make a GitHub release? Does the AWS pipeline automatically change cloudflare records, things like that?

Reply:
Good question.
The [`environment`](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment) attribute in GitHub Actions doesn't have any impact in our current infrastructure, it acts as a visual reference of where the deploy is going.

Consider this line as an informative piece of the workflow.

![image](https://user-images.githubusercontent.com/3170758/152197835-cbcc2859-b80d-49d2-b841-3a0fd182ba7f.png)

We are still stick in our original proposal to transition the `search` module in the moment that you consider appropriate.

Reply:
But @rbadillap what are the implications of this line:

`url: https://search.openverse.engineering`

since that is our production domain? What does this line do?

Reply:
@zackkrida that line displays the url as GitHub does in the image above

_taken from the [official documentation](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment)_

> You can also specify a URL for the environment. The specified URL will appear on the deployments page for the repository (accessed by clicking Environments on the home page of your repository) and in the visualization graph for the workflow run. If a pull request triggered the workflow, the URL is also displayed as a View deployment button in the pull request timeline.

Reply:
Oooh, I understand now. Thank you! So can we change this to `search-prod.openverse.engineering` then?

Reply:
Ah yeah, we can use the URL you want, do you want me to use `search-prod.openverse.engineering` for production and `search-staging.openverse.engineering` for staging?

Reply:
> Ah yeah, we can use the URL you want, do you want me to use `search-prod.openverse.engineering` for production and `search-staging.openverse.engineering` for staging?

done in the last commit